Document how to use develop_git (#22)
Varying-Vagrant-Vagrants/custom-site-template-develop: README.md

- A subdomain multisite
- A subdirectory multisite

> ## Convert code from SVN to Git

Reviewer: ```suggestion
## Switching from SVN to Git
```

>   vagrant ssh

Reviewer: If we use triple back ticks we can hint which syntax highlighting to use, e.g.

```
```shell
```

  cd /srv/www/[wordpress-trunk] # the folder name 
  ./bin/develop_git
  

> Run this commands let you to convert from svn to git and will require some time to download everything.

Reviewer: ```suggestion
Running this command will convert am svn based checkout to git and will require some time to run.
```

- A subdomain multisite
- A subdirectory multisite

## Switching from SVN to Git

```shell
vagrant ssh
cd /srv/www/[wordpress-trunk] # the folder name 
./bin/develop_git
```

> Running this command will convert am svn based checkout to git and will require some time to run.

Reviewer: ```suggestion
Running this command will convert an svn based checkout to git and will require some time to run.
```
